### PR TITLE
Martin D. - TO-DO List App

### PR DESCRIPTION
**Pull request by Martno96**
Link to my deployed project:
https://stoic-dubinsky-6740bc.netlify.app/

My reflections on how this week's project turned out:
I didn't have time for much this week, but I like what I managed to make, especially considering that I used a lot of time to learn the Material UI library

Things I'd like to have clarified or explained in more detail:
Nothing in particular, no :)

### CODE REVIEW

File: code/src/components/Task.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          <IconButton
+            onClick={() => dispatch(todoList.actions.removeTask(props.task.id))}
+          >
+            <DeleteIcon />
+          </IconButton>
+      </div>
+    </Box>
+  )
+}
+
```
Comment (rubmaigut) [inline code/src/components/Task.js:55]: very clean component. good job

File: code/src/components/TaskForm.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react"
+import { useDispatch } from "react-redux"
+import uniqid from "uniqid"
+import { TextField, Button } from "@material-ui/core"
+import AddIcon from "@material-ui/icons/Add"
+
+import todoList from "../reducers/todoList"
+
+
```
Comment (rubmaigut) [inline code/src/components/TaskForm.js:9]: I think that maybe you could use fewer empty lines here

File: code/src/components/TaskStats.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import { Box, AppBar } from "@material-ui/core"
+import { positions, sizing } from "@material-ui/system"
+import { useSelector } from "react-redux"
+
+
```
Comment (rubmaigut) [inline code/src/components/TaskStats.js:6]: is the same in the last comment. if you use ESlint lib, it might mark this as error

**Review by rubmaigut — APPROVED** [review]
Nice job, very clean code, I like the minimalistic design that you have implemented. keep going like that

File: code/public/style.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+}
+
+.task {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  margin: 6px 0px;
+  padding: 15px;
+  border-radius: 10px;
```
Comment (KarolCislowski) [inline code/public/style.css:40]: https://www.w3schools.com/cssref/css3_pr_word-wrap.asp will save the day I think:D 
image

File: code/src/App.js [hunk 5]
```diff
@@ -1,9 +1,40 @@
-import React from 'react'
+import React from "react"
+import { Provider } from "react-redux"
+import { combineReducers, configureStore } from "@reduxjs/toolkit"
+import ReactDOM from "react-dom";
+import { Box, Container, ThemeProvider } from "@material-ui/core"
```
Comment (KarolCislowski) [inline code/src/App.js:5]: when you import more than 2 components from the same library remember to organize them into a column. It is easier to read and debug 🤓 

```jsx
import { 
  Box,
  Container, 
  ThemeProvider } from "@material-ui/core"
```

File: code/src/theme.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    primary: { 
+      light: "#fff",
+      main: "#f7f7f7",
+      dark: "#d9d9d9",
+      contrastText: "#000"
+    },
+    secondary: { 
+      light: "#616161",
+      main: "#333333",
```
Comment (KarolCislowski) [inline code/src/theme.js:18]: you used short versions before, why not here?

**Review by KarolCislowski — APPROVED** [review]
Nice app. Please remember to fix the problem with word-wrapping 🤓

**Pull request closed without merging**